src/transformers/trainer.py

```diff
@@ -803,6 +809,14 @@ def train(
             self.optimizer, self.lr_scheduler = None, None
 
         # Load potential model checkpoint
+        if isinstance(resume_from_checkpoint, bool) and resume_from_checkpoint:
+            logger.info(
+                "resume_from_checkpoint passed as `True`. Will try loading last checkpoint from output directory ({args.output_dir})"
+            )
```

Comment:
I would put the log info after (in an else block after the error) to say "Resuming training from {resume_from_checkpoint}"

Reply:
Letting go of this log info since the next lines already have what you're looking for:  

>if resume_from_checkpoint is not None and os.path.isfile(os.path.join(resume_from_checkpoint, WEIGHTS_NAME)):
>logger.info(f"Loading model from {resume_from_checkpoint}).")